src/renderer/views/Playlist/Playlist.js

```diff
@@ -418,6 +423,20 @@ export default defineComponent({
       showToast(this.$t('User Playlists.SinglePlaylistView.Toast.This playlist does not exist'))
     },
 
+    checkPlaylist: function () {
+      if (
+        this.sortOrder === SORT_BY_VALUES.VideoLengthAscending ||
+        this.sortOrder === SORT_BY_VALUES.VideoLengthDescending
+      ) {
+        for (const v in this.playlistItems) {
+          if (!Object.hasOwn(v, 'lengthSeconds') || isNaN(v.lengthSeconds)) {
+            showToast(this.$t('User Playlists.SinglePlaylistView.Toast.This playlist has a video with a length error'), 10000)
+            break
+          }
+        }
```

Comment:
https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Array/some
I just prefer early return style if possible
```suggestion
        const anyVideoMissingLength = this.playlistItems.some(v => !Object.hasOwn(v, 'lengthSeconds') || isNaN(v.lengthSeconds))
        if (anyVideoMissingLength) {
          showToast(this.$t('User Playlists.SinglePlaylistView.Toast.This playlist has a video with a length error'), 10000)
        }
```